src/map.rs

```diff
@@ -290,6 +290,16 @@ impl<K, V> HashMap<K, V, DefaultHashBuilder> {
     /// The hash map is initially created with a capacity of 0, so it will not allocate until it
     /// is first inserted into.
     ///
+    /// Warning: `hash_builder` normally use a fixed key by default and that does
```

Comment:
Formatting nit: perhaps this should be under a subsection header? For example `# HashDoS resistance`.

Reply:
Done